### PR TITLE
Update/nodemailer massege

### PR DESCRIPTION
**Pull request by Jood80**
### Content
- [x]  The email content has been updated

### Testing
Write these following commands down in your terminal :
- pull origin update/nodemailerMassege
- npm run dev
- go to http://localhost:3000/reservation on your browser.
- After you fill all the information and submitting the form 
-  you should receive an email by now that has your reservation time and date

### UI
image

 **which is changed to be more peaceful as below** :point_down: 
image

### CODE REVIEW

File: server/helpers/nodeMailer.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ const nodemailer = require('nodemailer');
...
-  <ul>
-  <li>your reservation Date is: ${reservationDate}</li>
-  </ul>`;
+async function mailing(fullName, email, reservationDate, reservationTime) {
+  const htmlEmail = `<h3>${fullName} thank you for reserving in our Store..</h3>
+  <ul>your reservation Date is: ${reservationDate}- ${reservationTime}EST</ul>
+  <ul>Want to chat by phone to let us take your remains? Call us at (201) 330-1212</ul>
+  <ul>Monday – Saturday, 10:00am – 6:00pm EST.</ul>
+  <ul>Sundays, 12:00pm – 5:00pm EST </ul> 
+  `;
```
Comment (sara219) [inline server/helpers/nodeMailer.js]: Add a message that tells the customer, if they want to change or cancel their appointment to call by phone

**Review by sara219 — CHANGES_REQUESTED** [review]

File: server/helpers/nodeMailer.js (hunk 1 above)
Comment (Jood80) [reply server/helpers/nodeMailer.js]: yup, sure

**Review by Rahaf-96 — APPROVED** [review]
:muscle: :star_struck:

**Review by tasneembhiri — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Comment by okjake** [comment]
😂 looks good

**Pull request merged**